server/index.js

```diff
@@ -13,13 +13,14 @@ let templatize = require("./templatize");
 let Request = require("./request");
 let Security = require("./security");
 let localize = require("./localize");
-let errorhandling = require("./lib/errorhandling");
+const HttpError = require("./lib/http-error.js");
 let middleware = require("./lib/middleware")();
 let routes = require("./routes")();
-let getFileList = require("./lib/utils").getFileList;
+const Utils = require("./lib/utils");
 
 let server = express();
-let isDevelopment = env.get("NODE_ENV") === "development";
+const environment = env.get("NODE_ENV");
```

Comment:
yes, for example:

`let server = express()`

is server ever reassigned? probably not, so why isn't it const? it seems the current style in this file is using let for variable declaration.